### ISSUE TITLE
WCAG 1.4.3: Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (.tracking-tighter.text-muted-foreground\/70.text-\[10px\])

### ISSUE THREAD

**Issue by azap026** [azap026/smetalabv3#123]
#### Issue
`Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[Smetalab - Smart Engineering & Management](http://localhost:3000/app)
#### Element path
.border-border\/10.p-2\.5.rounded-lg:nth-child(1) > .gap-3.items-center.flex > .leading-tight > .tracking-tighter.text-muted-foreground\/70.text-\[10px\]
#### Snippet
`<p class="text-[10px] font-bold uppercase text-muted-foreground/70 tracking-tighter">PM</p>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 2.73 (foreground color: #9c9ca2, background color: #ffffff, font size: 7.5pt (10px), font weight: bold). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 144.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.46.0 (axe-core 4.10.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
